.github/workflows/ci.yml

```diff
@@ -57,12 +31,75 @@ jobs:
           --health-timeout 5s
           --health-retries 10
     steps:
-    - uses: actions/checkout@v3.5.0
-    - name: SBT
-      uses: ./.github/actions/sbt
-      env:
-        USE_DOCKER_FOR_TESTS: false
-        ES6_TEST_URL: http://elasticsearch:9200
-        LOCALSTACK_ENDPOINT: http://localstack:4566
-      with:
-        args: clean compile test:compile test
+      - uses: actions/checkout@v3
```

Comment:
There's also a later version of this - [v4](https://github.com/actions/checkout/releases), unless there's a specific reason to be using v3?